web/src/services/routing/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { lazy } from "react";
+import { BrowserRouter as Router, Navigate, Route, Routes, useParams } from "react-router-dom";
+
+import LoginPage from "@reearth/classic/components/pages/Authentication/LoginPage";
+import PasswordResetPage from "@reearth/classic/components/pages/Authentication/PasswordReset";
+import SignupPage from "@reearth/classic/components/pages/Authentication/SignupPage";
+import NotFound from "@reearth/classic/components/pages/NotFound";
+import AccountSettings from "@reearth/classic/components/pages/Settings/Account";
+import ProjectSettings from "@reearth/classic/components/pages/Settings/Project";
+import DatasetSettings from "@reearth/classic/components/pages/Settings/Project/Dataset";
+import PluginSettings from "@reearth/classic/components/pages/Settings/Project/Plugin";
+import PublicSettings from "@reearth/classic/components/pages/Settings/Project/Public";
+import SettingsProjectList from "@reearth/classic/components/pages/Settings/ProjectList";
+import WorkspaceSettings from "@reearth/classic/components/pages/Settings/Workspace";
+import AssetSettings from "@reearth/classic/components/pages/Settings/Workspace/Asset";
+import WorkspaceList from "@reearth/classic/components/pages/Settings/WorkspaceList";
+import { styled } from "@reearth/services/theme";
+
+import RootPage from "../../classic/components/pages/Authentication/RootPage";
+import Preview from "../../classic/components/pages/Preview";
+
+const BetaEditor = lazy(() => import("@reearth/beta/pages/Editor"));
+
+const EarthEditor = lazy(() => import("@reearth/classic/components/pages/EarthEditor"));
+const Dashboard = lazy(() => import("@reearth/classic/components/pages/Dashboard"));
+const GraphQLPlayground = lazy(() => import("@reearth/classic/components/pages/GraphQLPlayground"));
+const PluginEditor = lazy(() => import("../../classic/components/pages/PluginEditor"));
+
+export const AppRoutes = () => {
+  return (
+    <StyledRouter>
+      <Routes>
+        <Route index={true} element={<RootPage />} />
+        <Route path="login" element={<LoginPage />} />
+        <Route path="signup" element={<SignupPage />} />
+        <Route path="password-reset" element={<PasswordResetPage />} />
+        <Route path="dashboard/:workspaceId" element={<Dashboard />} />
+        {/* classic routes - start */}
+        <Route path="edit/:sceneId">
+          <Route index={true} element={<EarthEditor />} />
+          <Route path="preview" element={<Preview />} />
+        </Route>
+        <Route path="plugin-editor" element={<PluginEditor />} />
+        {/* classic routes - end */}
+        <Route path="scene/:sceneId" element={<BetaEditor />}>
+          <Route path="story" element={<BetaEditor />} />
+          <Route path="widgets" element={<BetaEditor />} />
+          <Route path="publish" element={<BetaEditor />} />
+        </Route>
+        <Route path="settings">
```

Comment:
Would need to add a line something like this to settings group:

```
<Route index={true} element={<Redirect to="account" />} />
```

When hit exact `/settings` then it seems not to match any route component.
Previously it redirected to `/settings/account` by https://github.com/reearth/reearth/pull/447/files#diff-663173f9ccb3067a584b11393e5923330cc0779256e7b47d6095cf11c50861e7L61

Reply:
Nice catch
[add index to /settings](https://github.com/reearth/reearth/pull/447/commits/9982ac6f46beb90724686677666ccd20aefeac77)